stable/sematext-agent/files/autodisco.yml

```diff
@@ -37,242 +28,167 @@
 # meta.
 #
 # labels:
-#    container.name: '{{if isset .ContainerInfo "Name"}}{{ .ContainerInfo.Name }}{{end}}'
-#    container.id:  '{{if isset .ContainerInfo "ID"}}{{ .ContainerInfo.ID }}{{end}}'
+#    container.name: '{{if isset .Meta.ContainerInfo "Name"}}{{ .Meta.ContainerInfo.Name }}{{end}}'
+#    container.id:  '{{if isset .Meta.ContainerInfo "ID"}}{{ .Meta.ContainerInfo.ID }}{{end}}'
 
 # You can also query app meta info through `Meta` field. For example, to get
 # the version of nginx server use '{{if isset .Meta "Version"}}{{ .Meta.Version }}{{end}}'
 # in `labels` section.
 
 cassandra:
-  process: java
-  args: org.apache.cassandra.service.CassandraDaemon
-  ports:
-    - 7199
-    - 7000
-    - 7001
-    - 9160
-    - 9042
+  name: Cassandra
+  signatures:
+    - java org.apache.cassandra.service.CassandraDaemon
 
 clickhouse:
-  process: clickhouse-.*
-  args: clickhouse-server --config=.*
-  ports:
-    - 8123
-    - 9000
-    - 9009
+  name: Clickhouse fast time-series storage engine
+  signatures:
+    - clickhouse-server
   labels:
     SPM_MONITOR_CLICKHOUSE_DB_HOST_PORT: "{{ .IP }}:8123"
     SPM_MONITOR_CLICKHOUSE_DB_USER:
     SPM_MONITOR_CLICKHOUSE_DB_PASSWORD:
 
 couchbase:
-  process: beam.smp
-  args: couchbase
-  ports:
-    - 8092
-    - 8093
-    - 8094
-    - 11210
+  name: Couchbase
+  signatures:
+    - beam.smp couchbase
   labels:
-    COUCHBASE_HOSTPORT: "{{ .IP }}:8093"
+    COUCHBASE_HOST_PORT: "{{ .IP }}:8093"
     COUCHBASE_QUERY_HOSTPORT: "{{ .IP }}:8093"
     COUCHBASE_BASICAUTH_USERNAME:
     COUCHBASE_BASICAUTH_PASSWORD:
 
 elasticsearch:
-  process: java
-  args: org.elasticsearch.bootstrap.Elasticsearch\b
-  ports:
-    - 9200
-    - 9300
-    - 9201
-    - 9202
-    - 9301
-    - 9302
+  name: Elasticsearch distributed search engine
+  signatures:
+    - java org.elasticsearch.bootstrap.Elasticsearch
   labels:
     SPM_MONITOR_ES_NODE_HOSTPORT: "{{ .IP }}:9200"
```

Comment:
This remains `HOSTPORT`?

Reply:
Yes. We decided to leave the conf variables for old integrations as-is